### PR TITLE
Support reading args from a config file

### PR DESCRIPTION
**Pull request by ducaale**
Ticks another item from #4.

Also, see https://httpie.io/docs#configurable-options

### CODE REVIEW

File: src/cli.rs [hunk 1]
```diff
@@ -33,6 +35,7 @@ use crate::{buffer::Buffer, request_items::RequestItem};
         AppSettings::DeriveDisplayOrder,
         AppSettings::UnifiedHelpMessage,
         AppSettings::ColoredHelp,
+        AppSettings::AllArgsOverrideSelf,
```
Comment (blyxxyz) [inline src/cli.rs:38]: Good catch, I didn't realize we didn't already have this behavior.

**Review by blyxxyz — CHANGES_REQUESTED** [review]

File: src/cli.rs (hunk 1 above)
Comment (ducaale) [reply src/cli.rs:38]: I also didn't know a flag like this existed until I read https://github.com/clap-rs/clap/discussions/2627

File: src/main.rs [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#![allow(clippy::bool_assert_comparison)]
+#![allow(clippy::needless_borrow)]
```
[the remark this answers is not in the document]
Comment (ducaale) [reply src/main.rs]: You are right. At first, I didn't properly understand what the warnings were about, so I disabled the `bool_assert_comparison` rule. Now I've re-enabled it and fixed the corresponding lint errors.

**Review by blyxxyz — APPROVED** [review]
LGTM

**Comment by blyxxyz** [comment]
Should this have gone into `master`?

**Comment by ducaale** [comment]
Nope, I will revert the recent changes on master and merge to develop instead 👍

**Pull request merged**